### PR TITLE
Add an atom feed for /stream

### PR DESCRIPTION
**Pull request by seanh**
This is https://github.com/hypothesis/h/pull/2029 rebased

### CODE REVIEW

File: h/api_client/api_client.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        :rtype: dict or list
+
+        :param path: The API path to request, for example: ``"/search"``.
+
+        :param params: The params to pass in the URL's query string,
+            for example: ``{"limit": 10}``.
+        :type params: dict
+
+        """
+        url = "/".join([self.base_url.rstrip("/"), path.lstrip("/")])
```
Comment (tilgovi) [inline h/api_client/api_client.py]: Clearer, I think: `urlparse.urljoin(self.base_url, path)`

Comment (tilgovi) [inline h/api_client/api_client.py]: Except then the `path` should not start with a `/` or it'll be relative to the domain root.

**Comment by tilgovi** [comment]
:cake:

**Comment by tilgovi** [comment]
@seanh's first PR?

**Comment by seanh** [comment]
[Second](https://github.com/hypothesis/h/pull/2056)

**Comment by tilgovi** [comment]
Oops. I missed the first celebration opportunity. Anyway, \o/

**Pull request merged**